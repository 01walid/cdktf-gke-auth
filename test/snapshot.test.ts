import { Testing } from "cdktf";
import "cdktf/lib/testing/adapters/jest"; // Load types for expect matchers

import { GKETestingStack } from "./stack";

describe("Snapshot testing", () => {
  let stack: GKETestingStack;
  let synthedOutputJSON: string;

  beforeAll(() => {
    const app = Testing.app();
    stack = new GKETestingStack(app, "testing-stack");
    synthedOutputJSON = Testing.synth(stack);
  });

  it("synths successfully", () => {
    expect(synthedOutputJSON).toMatchSnapshot();
  });
});
