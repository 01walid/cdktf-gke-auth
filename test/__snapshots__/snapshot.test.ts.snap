// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing synths successfully 1`] = `
"{
  "data": {
    "google_client_config": {
      "gke-auth_client-config_BA8E1446": {
      }
    },
    "google_container_cluster": {
      "gke-auth_gke-cluster_6C118177": {
        "location": "europe-west1",
        "name": "my-cluster",
        "project": "my-project"
      }
    }
  },
  "output": {
    "clusterCaCertificate": {
      "sensitive": true,
      "value": "\${data.google_container_cluster.gke-auth_gke-cluster_6C118177.master_auth[0].cluster_ca_certificate}"
    },
    "host": {
      "value": "https://\${data.google_container_cluster.gke-auth_gke-cluster_6C118177.endpoint}"
    },
    "token": {
      "sensitive": true,
      "value": "\${data.google_client_config.gke-auth_client-config_BA8E1446.access_token}"
    }
  },
  "provider": {
    "google": [
      {
      }
    ]
  },
  "terraform": {
    "required_providers": {
      "google": {
        "source": "google",
        "version": "4.62.0"
      }
    }
  }
}"
`;
